website/src/components/demos/geo-layer-demos.js

```diff
@@ -73,29 +73,9 @@ export const TileLayerDemo = createLayerDemoClass({
   formatTooltip: f => JSON.stringify(f.properties),
   allowMissingData: true,
   props: {
-    stroked: false,
     opacity: 1,
-
-    getLineColor: [192, 192, 192],
-    getFillColor: [140, 170, 180],
-
-    getLineWidth: f => {
```

Comment:
Why do we need to remove this? The demo will end up not matching the sample code in the documentation.

Reply:
Ok, fixed it.
Problem was, `propType` for these props was undefined causing a crash. Fixed it in `layer-params.js`.
But side question: in `tile-layer.js` should we change

```
const defaultProps = {
  renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props)},
  ...
  maxCacheSize: null
};
```
to

```
const defaultProps = Object.assign( {}, GeoJsonLayer.defaultProps,  {
  renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props)},
  ...
  maxCacheSize: null
};

```

Reply:
No, because `TileLayer` is not required to use GeoJsonLayer as its sublayer.